packages/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter
-version: 0.5.22+1
+version: 0.5.22+2
```

Comment:
Since we are adding a new API, we should bump the minor version: 0.5.23 
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
This has been fixed considering the changes for federated implementation style.